src/main/resources/com/mathworks/ci/TestResultsViewAction/index.jelly

```diff
@@ -0,0 +1,217 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:st="jelly:stapler" xmlns:c="/charts">
+    <l:layout title="MATLAB Test Results">
+        <l:side-panel>
+            <st:include page="sidepanel.jelly" it="${it.build}" optional="true" />
+        </l:side-panel>
+
+        <l:app-bar title="MATLAB Test Results">
+            <button class="jenkins-button" onclick="window.open('https://matlab.mathworks.com/open/github/v1?repo=${GIT_URL}')" tooltip="View source in MATLAB Online">Open in MATLAB Online</button>
+            <t:help href="https://github.com/jenkinsci/matlab-plugin/blob/master/CONFIGDOC.md" tooltip="MATLAB Plugin Configuration Guide" />

```

Comment:
I think it would be valuable to keep "MATLAB" in the heading as these are just MATLAB test results, the user might be running other tests in their job whose results we won't be showing here.

Reply:
I think "MATLAB Test Results" is fine. My comment was about "MATLAB Plugin Configuration Guide."

Reply:
Sure, that makes sense! Updated